docs/reference/configurations.mdx

```diff
@@ -114,6 +114,8 @@ your website. The term website refers to what a
   - `intersectionObserverDevices`: the type of devices where intersection
     observers should be used for preload. We usually don't want it on desktop to
     avoid triggering too many requests. (default: `["phone", "tablet"]`)
+- `hasDefaultAddressSupport`: a boolean that controls if the customer can set
+  addresses as default for billing and shipping
```

Comment:
Looking at the [addition in the MR](https://gitlab.blackswift.cloud/front-commerce/front-commerce/-/blob/fd27e93d3e345157251ac6d3ace5e5e56c528ce6/src/web/theme/modules/User/Address/AddressForm/AddressForm.js#L117-119), this seems to default to `true` I would be good to note it as such in the docs.

```suggestion
- `hasDefaultAddressSupport`: a boolean that controls if the customer can set
  addresses as default for billing and shipping (default: `true`)
```